handout/features/typings.md

```diff
@@ -3,9 +3,7 @@
 Astute readers might be wondering what happens when TypeScript programmers need
 to interface with JavaScript modules that have no type information.  TypeScript
 recognizes files labelled `*.d.ts` as _definition_ files.  These files are
-meant to use TypeScript to describe interfaces presented by JavaScript 
+meant to use TypeScript to describe interfaces presented by JavaScript
 libraries.
 
-There are communities of people dedicated to creating typings for JavaScript
```

Comment:
@watrool are there still some legacy use cases for `typings`?  If so we might want to keep this and be clear that it's deprecated but still exists for legacy reasons.